pipeline/pipeline.go

```diff
@@ -273,7 +273,7 @@ func (p *Pipeline) In(sourceID SourceID, sourceName string, offset int64, bytes
 	}
 
 	event := p.eventPool.get()
-	dec := decoder.NO
+	var dec decoder.DecoderType
```

Comment:
Does the previous version look more readable?

Reply:
linter doesn't like it - ineffective assignment